app/views/facility_journals/_problem_transactions.html.haml

```diff
@@ -0,0 +1,54 @@
+.row
+  .span12
+    %p.alert.alert-error= t("facility_journals.new.expired_payment")
+.row
+  .span12
+    %table.table.table-striped.table-hover.dense.old-table
+      %thead
+        %tr
+          %th= Order.model_name.human
+          %th= OrderDetail.model_name.human
+          %th.nowrap= OrderDetail.human_attribute_name(@date_range_field)
+          - if current_facility.blank? || cross_facility?
+            %th= Facility.model_name.human
+          %th.order-note= OrderDetail.human_attribute_name("description")
+          %th.nowrap= OrderDetail.human_attribute_name("user")
+          - unless @account
```

Comment:
I don't think we need to worry about this conditional. I think in this scenario it will always want to render.